tensorboard/plugins/scalar/vz_line_chart/vz-chart-helpers.ts

```diff
@@ -41,6 +41,12 @@ export interface Point {
   dataset: Plottable.Dataset;
 }
 
+export interface TooltipColumn {
```

Comment:
I like the declarative nature of this approach—specifying a list of tooltip columns—and think that it is a good way to go.

That said, I think that `computeFunction` deserves a better name. Depending on how you parse the current name, it's either misleading (it does not compute a function) or redundant (anything that computes must be a function). Suggest something like `evaluate` (or `compute`). As always, consider how it reads at the call site: `columns[i].evaluate(datum)` sounds best, imho.